engine/src/test/scala/cromwell/SimpleWorkflowActorSpec.scala

```diff
@@ -107,7 +107,7 @@ class SimpleWorkflowActorSpec extends CromwellTestKitWordSpec with BeforeAndAfte
     }
 
     "fail to construct with inputs of the wrong type" in {
-      val expectedError = "Could not coerce JsNumber value for 'wf_hello.hello.addressee' (3) into: WdlStringType"
+      val expectedError = "No coercion defined from '3' of type 'spray.json.JsNumber' to 'String'."
```

Comment:
it would have been nice to keep the name of the offending input here :/ I probably unintentionally changed that in the last wdl4s PR 😞